modules/cudf/src/types/enums.ts

```diff
@@ -20,3 +20,10 @@ export enum NullOrder
   AFTER,
   BEFORE
 }
+
+export enum DuplicateKeepOption
+{
+  first,  // Keeps first duplicate row and unique rows.
+  last,   // Keeps last duplicate row and unique rows.
+  none,   // Keeps only unique rows are kept.
```

Comment:
Do we want to capitalise these as with `NullOrder`?

Reply:
I'd vote we make the `NullOrder` keys not screaming case, but that's a personal pref.

Reply:
^ that works as well 👍

Reply:
changed case of `NullOrder` values in ee94e25

Reply:
whoops. that introduced some merge conflicts. let me fix that.

Reply:
fixed.